sycl/source/detail/program_manager/program_manager.cpp

```diff
@@ -2197,8 +2197,9 @@ bool doesDevSupportDeviceRequirements(const device &Dev,
 
   auto AspectsPropIt = getPropIt("aspects");
   auto ReqdWGSizePropIt = getPropIt("reqd_work_group_size");
+  auto ReqdSubGroupSizePropIt = getPropIt("reqd_sub_group_size");
 
-  if (!AspectsPropIt && !ReqdWGSizePropIt)
+  if (!AspectsPropIt && !ReqdWGSizePropIt && !ReqdSubGroupSizePropIt)
```

Comment:
nit: I'd remove these two lines alltogether.

Reply:
Agree, this is redundant